.github/workflows/curldemo.yaml

```diff
@@ -0,0 +1,29 @@
+name: curldemo
+
+on:
+  push:
+    paths:
+      - 'deploy/test-apps/curldemo/**'
+
+jobs:
+  curldemo:
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_CLI_EXPERIMENTAL: "enabled"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
```

nit: `fetch-depth: 0` [indicates](https://github.com/actions/checkout/blob/ec3a7ce113134d7a93b817d10a8272cb61118579/README.md?plain=1#L92) that all history, branches and tags will be fetched, but we are not doing any fancy git stuff here, so the default shallow fetch should be ok. If omitted the `1` is used